src/common/actions/repositories.js

```diff
@@ -3,6 +3,7 @@ import 'isomorphic-fetch';
 import { conf } from '../helpers/config';
 
 const BASE_URL = conf.get('BASE_URL');
+const log = console;
```

Comment:
Get around console statement linting.

Correct me if I'm wrong, can we not use winston here?